Task/ahh 111

tim0thysun opened the pull request:
This completes the welcome screen component, which should be displayed after a user authenticates with their UW net-IDs.

tim0thysun committed:
welcome component

tim0thysun committed:
welcome page done

KeithSRoberts commented on foodalert/static/foodalert/js/components/welcome-template.vue:
```diff
@@ -0,0 +1,67 @@
+  export default {
+    data() {
+      return {
+        mainProps: { width: 40, height: 27 } // UW Block Logo has aspect ratio of 150:101
```
This may not need to be a prop if we dont expect the size of the UW Block Logo to change, it could instead be a hard coded value.

tim0thysun replied:
Right, I'll actually download the image into the img folder and use it from there.

KeithSRoberts commented on foodalert/static/foodalert/js/components/welcome-template.vue:
```diff
@@ -0,0 +1,67 @@
+
+<style lang="scss">
+    @import '../../css/_mixins.scss';
+    .hh-logo {
```
What is using the '.hh-logo' class? I can tell it's outline changes color based on a particular media query, but I cant seem to find where this change occurs

tim0thysun replied:
Will remove this—used primarily to test different breakpoints.

KeithSRoberts commented on foodalert/static/foodalert/js/components/welcome-template.vue:
```diff
@@ -0,0 +1,67 @@
+      }
+      @include breakpoint(desktop) {
+       outline-color: blue;
+       // padding-left: 15px !important;
```
Its best to remove comments if they were used in testing or are no longer needed so that they dont get accidentally used in the future

tim0thysun replied:
Will do.

KeithSRoberts commented on foodalert/static/foodalert/js/components/welcome-template.vue:
```diff
@@ -0,0 +1,67 @@
+       // padding-right: 15px !important;
+      }
+    }
+    .hh-back-button {
```
Likewise for this scss property, I'm not sure which button this changes

tim0thysun replied:
This was also for testing things out. Will remove.

KeithSRoberts commented on foodalert/static/foodalert/js/base.js:
```diff
@@ -13,6 +13,8 @@ window.vueData.components = {
 };
 
 window.vueData.routes = [
+  { path: "/welcome", component: UnauthzTemplate},
```
Is the idea that we want this to be the first page all users hit? If so, it should still probably be a child route our Generic Template, but we may need to discuss the flow/ how this page is meant to be visited in person.

tim0thysun replied:
I talked to Adi about this and we put it outside the Generic Template because it got rid of the empty heading bar at the top.

Commit:
Updates to welcome component

Made changes requested by Keith for pull request. Only thing that did not change was the component not being a child route of Generic Template.

tim0thysun committed:
updates

tim0thysun committed:
Update welcome-template.vue

tim0thysun committed:
Merge branch 'develop' into task/AHH-111

tim0thysun committed:
Update welcome-template.vue

tim0thysun committed:
Merge branch 'develop' into task/AHH-111

tim0thysun committed:
updates
